fp.js

```diff
@@ -18,11 +23,16 @@ var people = [
   {name: "Jessica", age: 34},
   {name: "Jasmine", age: 39}
 ]
-people.forEach(person => {
-  if(person.age < 35){
-    peopleYoungerThan35.push(person)
-  }
-})
+// people.forEach(person => {
+//   if(person.age < 35){
+//     peopleYoungerThan35.push(person)
+//   }
+// })
+
+function youngerThan(age){
+  return age < 35 ;
+}
+var filtered =people.filter(youngerThan);
 
```

Comment:
again, very close...`age` is a property on each individual `person`, so you would need to change your `youngerThan` function to something like: 
```js
function youngerThan (person) {
 return person.age < 35
}
```